widgetastic_manageiq/__init__.py

```diff
@@ -1089,6 +1091,119 @@ def workaround_save_issue(self):
             "{}._handlers.change.map(function(handler) {{ handler() }});".format(self.item_name))
 
 
+class SSUIVerticalNavigation(VerticalNavigation):
+    """The Patternfly Vertical navigation."""
+    CURRENTLY_SELECTED = './/li[contains(@class, "active")]/a/span'
+
+    @property
+    def currently_selected(self):
+        for el in self.browser.elements(self.CURRENTLY_SELECTED, parent=self):
+            return self.browser.text(el)
+
+
+class SSUIlist(Widget, ClickableMixin):
+    """Represents the list of items like Services ,
+    Orders or service catalogs in SSUI pages.
+    """
+
+    LOCATOR = '//div[@id={}]/../div//a[normalize-space(.)={}]'
+
+    def __init__(self, parent, list_name, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.list_name = list_name
+
+    def locator(self, item_name):
+        return self.LOCATOR.format(quote(self.list_name), quote(item_name))
```

Comment:
PROBABLY REQUIRED: This should be probably converted to ParametrizedView, which would probably be better suited for this kind of use.

Reply:
Done